scripts/zones/Ceizak_Battlegrounds/mobs/Unfettered_Twitherym.lua

```diff
@@ -0,0 +1,110 @@
+local entity = {}
```

Comment:
This needs the standard header, also with:

This will also need comments with commands for `!pos` (take you to the NM's spawn point), and `!additem` or some sort of `!exec` to force-pop the NM, for easier testing